docs/compute/git.md

```diff
@@ -0,0 +1,152 @@
+# Git integration 
+
+The Git integration allows users to manage and version control their feature engineering codebase and workflows using Git directly from the Hopsworks UI. The integration currently supports repositories hosted on Github, Gitlab and Bitbucket. 
```

Comment:
```suggestion
The Git integration allows users to manage and version control their codebase and workflows using Git directly from the Hopsworks UI. The integration currently supports repositories hosted on Github, Gitlab and Bitbucket. 
```

Reply:
True for now, but the focus is shifting towards an e2e platform so the codebase is not strictly feature engineering.